proto/feature/service.proto

```diff
@@ -301,6 +301,7 @@ message EvaluateFeaturesRequest {
   bucketeer.user.User user = 1;
   string environment_namespace = 2;
   string tag = 3;
+  string feature_id = 4;  // it will evaluate a single feature if set
```

Comment:
Can you explain the reason why we need to handle single feature instead of creating new API such as EvaluateFeature?

Reply:
The main reason is to avoid code duplication.
Also, this API is for internal services only.